Testscripts/Linux/XDPDumpSetup.sh

```diff
@@ -110,6 +112,25 @@ function Run_XDPDump {
     fi
 }
 
+# Check if kernel supports XDP or not
+function kernel_check {
+    if [ -z "${1}" -o -z "${2}" ]; then
+        LogErr "ERROR: must provide install ip and NIC Name to Run_XDPDump"
+        SetTestStateAborted
+        exit 1
+    fi
+    local install_ip="${1}"
+    local nic_name="${2}"
+    command="ethtool -S ${nic_name}  | grep xdp_drop | wc -l"
+    xdp_counter="$(ssh ${install_ip} $command)"
+    if [ $xdp_counter -gt 0 ]; then
+        LogMsg "Kernel version supports XDP"
+    else
+        LogErr "Kernel Version does not support XDP"
+        SetTestStateSkipped
+        exit 1
```

Comment:
Why exit with 1 ?

Reply:
to indicate general error. Is there any issue with it?

Reply:
As per design test case should always return 0 after setting appropriate state.